Makefile

```diff
@@ -17,6 +17,10 @@ docker-test:
 	${CONTAINER_ENGINE} build . -f $(OPERATOR_DOCKERFILE).test -t $(OPERATOR_IMAGE_URI_TEST)
 	${CONTAINER_ENGINE} run -t $(OPERATOR_IMAGE_URI_TEST)
 
+.PHONY: e2e-test
+e2e-test:
+	ginkgo run test/e2e/
```

Comment:
I think it would nice to mention this in the README file including some link to ginkgo installation. WDYT?